docs/user/concepts/measurement.rst

```diff
@@ -0,0 +1,71 @@
+.. _dismod-measurement:
+
+Measurement, Rate, Integrand
+----------------------------
+
+The Dismod-AT program has
+`its own documentation <https://bradbell.github.io/dismod_at/doc/dismod_at.htm>`_,
+which serves well for specifics about database tables, definitions of distributions,
+and other details. This documentation is a high-level view of what Dismod-AT does
+in order to explain what you can do with the Cascade.
+
+Dismod-AT does statistical estimation. It is a nonlinear, multi-level regression.
+The two hierarchical levels are the measurements, at the micro level, and the locations,
+at the macro level.
+
+.. index:: measurement
+
+*Measurements* are input data from data bundles. Every measurement has
+a positive, non-zero standard deviation. A measurement may or may not have
+the same upper and lower age or the same upper and lower time.
+All measurements are associated with locations.
+
+Dismod-AT's central feature is that it estimates rates of a disease process.
+The disease process is nonlinear and described by a
+`differential equation <https://bradbell.github.io/dismod_at/doc/avg_integrand.htm>`_.
+We can discuss the behavior of that model in detail later. For this differential
+equation,
+
+ 1. Rates go in.
+ 2. Prevalence and death comes out.
+
+.. index:: rate
+
+A *Rate* is incidence, remission, excess mortality, other-cause mortality, or
+initial prevalence. A rate is a continuous function of age and time. It's specified
+as a set of points, and interpolated between those points, but it's continuous.
+Even the initial prevalence is continuous across time but defined only for the youngest
+age. The data associated with rates is defined at points of age and time, so
+it isn't associated with age or time ranges. It also doesn't have standard
+deviations.
+
+If we think of a typical linear regression,
+
+.. math::
+
+    y = a + bx + \epsilon
+
+we can draw an equivalence for Dismod-AT where :math:`x` are the covariates,
+:math:`b` are the covariate multipliers, :math:`\epsilon` are distributions of priors,
+`a` are the rates, and `y` are the observations. How Dismod-AT connects
+rates to observations is much more complicated than a typical linear regression.
+
+In order to relate a rate to an observation, Dismod-AT has to do a
+few steps.
+
+ 1. Use the ODE to predict prevalence and death.
```

Comment:
It's not a reference to those programs. ODEs are a general concept.

Reply:
This is the same `ordinary differential equation` that is the ODE in DismodODE. And it's used, at least at a high level, for the same thing in DismodeODE as in DismodAT. It defines the mathematical relationships between the primary rates and those relationships define the space of possible values that the rates can have. The ODE is what allows dismod to know whether some combination of values for the rates are consistent with each other. Without that constraint the estimates wouldn't be guaranteed to be epidemiologicaly plausible.

Reply:
When Theo or Chris point at a screen at say things like "this graph must slope down because that other graph has a bump here and this third graph is flat" they're using intuition about the properties of the ODE dismod uses to make those judgments.